demo/admin.py

```diff
@@ -5,6 +5,10 @@
 
 from .models import Person, FavoriteColor, Tag, PersonTag
 
+# Below, importing unused class from autocomplete.py to trigger
+# the registering of agnocomplete models
+from .autocomplete import logger
```

Comment:
When you need to import a class but it's unused, a good way to avoid flake8 errors is to defin it in __all__

Reply:
@ewjoachim meaning, you import it and then a few lines below you define: `__all__ = ['imported_class']`?

Reply:
yes. Flake8 has a rule saying that it's not considered unused if it appears in the `__all__`.